jsonschema-module-jakarta-validation/src/main/java/com/github/victools/jsonschema/module/jakarta/validation/JakartaValidationModule.java

```diff
@@ -499,6 +504,27 @@ protected BigDecimal resolveNumberExclusiveMaximum(MemberScope<?, ?> member) {
         return null;
     }
 
+    /**
+     * Look-up the finite list of possible values.
+     * @param member field/method to determine allowed values for
+     * @return applicable "const"/"enum" values or null
+     * @see AssertTrue
+     * @see AssertFalse
+     */
+    protected List<Object> resolveEnum(MemberScope<?, ?> member) {
+        List<Object> values = new ArrayList<>();
+
+        if (this.getAnnotationFromFieldOrGetter(member, AssertTrue.class, AssertTrue::groups) != null) {
+            values.add(true);
+        }
+
+        if (this.getAnnotationFromFieldOrGetter(member, AssertFalse.class, AssertFalse::groups) != null) {
+            values.add(false);
+        }
+
+        return values.isEmpty() ? null : values;
```

Comment:
**suggestion**: As per my other comment, I'd deem it an invalid combination if something is supposed to be both `true` and `false` at once.
Therefore, we could adjust this here slightly to reduce the overall complexity.
```suggestion
        List<Object> values;
        if (this.getAnnotationFromFieldOrGetter(member, AssertTrue.class, AssertTrue::groups) != null) {
            values = Collections.singletonList(true);
        } else if (this.getAnnotationFromFieldOrGetter(member, AssertFalse.class, AssertFalse::groups) != null) {
            values = Collections.singletonList(false);
        } else {
            values = null;
        }
        return values;
```
That would have an impact on your excellent unit tests of course.

Reply:
Do you think it'd be better if an exception was thrown if someone tried setting both `@AssertTrue` and `@AssertFalse`? Instead of silently ignoring `@AssertFalse`?

Reply:
Combining `@AssertTrue` and `@AssertFalse` is a programming error, as the validation always fails.
Technically, I'd ignore it then as both are probably meant to be allowed.
However, adding logic for such a "programming error" seems unnecessary.

Therefore it's okay to assume that this error/mistake won't exist (or will be fixed independently) and keep the logic simple here as per my above suggestion.
I think It's not worth adding extra logic (neither to ignore invalid annotations nor throwing an error during the schema generation) if we assume this scenario will never occur.
In case somebody happens to raise this as a bug in the future, I'd be very curious about the justification for having both at the same time. 😉